src/Relation/RefersTo.php

```diff
@@ -61,24 +72,16 @@ public function queue(CC $store, $entity, Node $node, $related, $original): Comm
             $this->getSource($node->getRole())->getTable()
         );
 
-        // fastest way to identify the entity
-        $pk = $this->orm->getSchema()->define($node->getRole(), Schema::PRIMARY_KEY);
+        $this->forwardContext($rNode, $this->outerKeys, $update, $node, $this->innerKeys);
+        if ($store instanceof Insert) {
```

Comment:
I think there is an interface for that command... The actual command name should not be here.